tests/unit/json/pull_review_example

```diff
@@ -0,0 +1,36 @@
+{
+  "id": 80,
+  "user": {
+    "login": "octocat",
+    "id": 1,
+    "avatar_url": "https://github.com/images/error/octocat_happy.gif",
+    "gravatar_id": "",
+    "url": "https://api.github.com/users/octocat",
+    "html_url": "https://github.com/octocat",
+    "followers_url": "https://api.github.com/users/octocat/followers",
+    "following_url": "https://api.github.com/users/octocat/following{/other_user}",
+    "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
+    "organizations_url": "https://api.github.com/users/octocat/orgs",
+    "repos_url": "https://api.github.com/users/octocat/repos",
+    "events_url": "https://api.github.com/users/octocat/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/octocat/received_events",
+    "type": "User",
+    "site_admin": false
+  },
+  "body": "Here is the body for the review.",
+  "commit_id": "ecdd80bb5713475d7ba9641ffaa4d7d2c19d3f3091",
+  "state": "PENDING",
+  "html_url": "https://github.com/octocat/Hello-World/pull/1347#pullrequestreview-80",
+  "pull_request_url": "https://api.github.com/repos/octocat/Hello-World/pulls/1347",
+  "author_association": "CONTRIBUTOR",
```

Comment:
Documentation suggests that `author_association` is optional, but added it into this example after inspecting the returned json. Not sure how to confirm one way or the other?

Reply:
I've found that the API documentation doesn't actually keep up with the developers. This was better when the docs were F/OSS and people could contribute fixes/updates/etc but that's no longer the case and the documentation is increasingly un-trustworthy. I've found attributes documented that I have never seen in a real API response :(

Reply:
Inspecting a few other review comments indicates that it is returned even in the case where the author has no association with the project, you'll just see the following
```
"author_association": "NONE",
```

Possibly they realised it was easier to always return it instead of making it optional.

Reply:
Who knows. Thank you for pointing this out though. It's incredibly helpful to have that mentioned somewhere. 👍